ts/src/flexible-event/main.ts

```diff
@@ -114,7 +115,8 @@ if (options.json_file !== undefined) {
     throw new Error('windows and buckets must have same length')
   }
   const attributionScopes: AttributionScopes | null =
-    options.attribution_scope_limit === undefined
+    options.attribution_scope_limit === undefined ||
```

Comment:
Shall we add check that the limit has to be non-null if max event states is set?